apps/mainsite/templates/email/base.html

```diff
@@ -204,7 +204,7 @@
                                   <tr>
                                     <td align="center" style="font-size:0px;padding:10px 25px;word-break:break-word;">
                                       <div style="font-family:'Open Sans', Helvetica, Arial, sans-serif;font-size:10px;font-weight:400;line-height:14px;text-align:center;color:#aaaaaa;">
-                                        Or, you can copy/paste this link into your browser: <br>
+                                        Alternativ können Sie diesen Link kopieren und in Ihren Browser einfügen: <br>
```

Comment:
We are using "du" and not "Sie" in most other cases. We should stick to one.

Reply:
I updated it  to `du`.